src/modules/json.c

```diff
@@ -1,35 +1,9 @@
 #include "json.h"
-#include "utf8.h"
```

Comment:
Is this not needed now then?

Reply:
was only used in the escape from C for the utf8compare function.
but the file itself contains numerous string utilities that may be used for strings.wren

Reply:
Fine, we can leave it for now.